com.unity.render-pipelines.universal/Runtime/2D/Renderer2D.cs

```diff
@@ -176,6 +176,11 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
 
             ConfigureCameraTarget(colorTargetHandle.Identifier(), depthTargetHandle.Identifier());
 
+            // Add passes from Renderer Features.
+            isCameraColorTargetValid = true;    // This is to make it possible to call ScriptableRenderer.cameraColorTarget in the custom passes.
+            AddRenderPasses(ref renderingData);
+            isCameraColorTargetValid = false;
```

Comment:
I'd advise against allowing to call cameraColorTarget in AddRenderPasses if you ever have some logic to decide to draw to staging texture or backbuffer. That's why we prevent that in our renderer. We can't know before we figure pass dependencies.